google.json

```diff
@@ -1,13 +1,200 @@
 [
+    {
+        "id": "com.google.android.turboadapter",
+        "label": "Device Health Services Adapter",
+        "description": "Another app for Device Health Services(discontinued)",
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.apps.security.securityhub",
+        "description": "Security Hub",
+        "web": [
+            "https://play.google.com/store/apps/details?id=com.google.android.apps.security.securityhub"
+        ],
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.audio.hearing.visualization.accessibility.scribe",
+        "description": "Live Transcribe & Notification",
+        "web": [
+            "https://play.google.com/store/apps/details?id=com.google.audio.hearing.visualization.accessibility.scribe"
+        ],
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.pixel.setupwizard",
+        "label": "Pixel Setup",
+        "description": "Its only needed on first-boot setup",
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.captiveportallogin",
+        "label": "CaptivePortalLogin",
+        "description": "Its the same as(com.android.captiveportallogin). Support for captive portal.\nA captive portal login is a web page where the users have to input their login information or accept the displayed terms of use. \nSome networks (typically public wifi network) use the captive portal login to block access until the user inputs \nsome necessary information\nNOTE : This package is a now a mandatory mainline module",
```

Comment:
Description is in violation with `removal`. It says mandatory but removal says replaceable.